src/main/java/org/broadinstitute/hellbender/tools/spark/sv/GatherSplitReadsSpark.java

```diff
@@ -172,7 +286,9 @@ else if ( read.getContig() == read.getMateContig() )
         private static final int FUNKY_PAIR_LOCUS_WIDTH = 100; // uncertainty in event locus for discordant pair
```

Comment:
So we don't forget, put in a TODO comment here that we'll eventually want to make this depend on the fragment size distribution.